index.src.html

```diff
@@ -9,6 +9,7 @@
 Group: mediawg
 Editor: Paul Adenot, w3cid 62410, Mozilla https://www.mozilla.org/
 Editor: Bernard Aboba, w3cid 65611, Microsoft Corporation https://www.microsoft.com/
+Editor: Eugene Zemtsov, w3cid 139375, Google LLC https://www.google.com/
```

Comment:
Is it supposed to be LLC or Inc?

Reply:
This question also gave me a pause. I decided to use LLC based on:

1. Google is LLC now. https://en.wikipedia.org/wiki/Google
2. Other new spec editors seem to use LLC:
   -  https://www.w3.org/TR/hr-time-3/
   -  https://www.w3.org/TR/permissions/

Reply:
Per internal discussion Google or Google LLC are fine.